docs/platforms/javascript/guides/nuxt/install/dynamic-import.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Dynamic Import (default)
+sidebar_order: 1
+description: "Learn about how the Nuxt SDK leverages dynamic input() in the build output."
+---
+
+## Understanding the `import()` expression
+
+The `import()` expression (also called "dynamic import") allows conditional and flexible module loading in ESM.
+For the Sentry Nuxt SDK, it provides an approach to initialize server-side configuration before application startup.
```

Comment:
l/feel free to disregard: Since we're already in deep dive territory, we can explain _why_ the SDK needs to be initialized before the rest of the server code. WDYT?
 
```suggestion
For the Sentry Nuxt SDK, it provides an approach to initialize server-side configuration before application startup. This early initialization is required to set up the SDK's instrumentation of various libraries correctly.
```